inputfiles/overridingTypes.json

```diff
@@ -897,6 +897,18 @@
                     }
                 }
             },
+            "HTMLElement": {
+                "methods": {
+                    "method": {
+                        "focus": {
+                            "name": "focus",
+                            "override-signatures": [
+                                "focus(options?: FocusOptions): void"
```

Comment:
so why is this needed?

Reply:
For https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus. (Separate from nonce, though.)